typed_ast/__init__.py

```diff
@@ -1 +1 @@
-
+__version__ = "1.1.0"
```

Comment:
I'm not trying to change the version as part of this PR -- versions bumps have their own commits.